<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWords">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="favButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>☆★</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>type your query here...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Results</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>Mode</string>
     </property>
     <addaction name="actionLightMode"/>
     <addaction name="actionDarkMode"/>
    </widget>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Query Language</string>
     </property>
     <addaction name="actionChooseLangEn"/>
     <addaction name="actionChooseLangFr"/>
     <addaction name="actionChooseLangPl"/>
     <addaction name="actionChooseLangEs"/>
    </widget>
    <addaction name="menuMode"/>
    <addaction name="menuLanguage"/>
    <addaction name="actionChangeName"/>
    <addaction name="actionDeleteUser"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionNewUser_2"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuOptions"/>
   <addaction name="menuOther"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewUser">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionChooseUser">
   <property name="text">
    <string>Guest</string>
   </property>
  </action>
  <action name="actionLightMode">
   <property name="text">
    <string>Light Mode</string>
   </property>
  </action>
  <action name="actionDarkMode">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionChooseLangEn">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChooseLangFr">
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionChooseLangPl">
   <property name="text">
    <string>Polish</string>
   </property>
  </action>
  <action name="actionChangeUsername">
   <property name="text">
    <string>Change Username</string>
   </property>
  </action>
  <action name="actionChooseLangEs">
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNewUser_2">
   <property name="text">
    <string>New User</string>
   </property>
  </action>
  <action name="actionGuest">
   <property name="text">
    <string>Guest</string>
   </property>
  </action>
  <action name="actionChangeName">
   <property name="text">
    <string>Change Username</string>
   </property>
  </action>
  <action name="actionDeleteUser">
   <property name="text">
    <string>Delete Current User</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>searchButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
